sdk/confidentialledger/azure-confidentialledger/azure/confidentialledger/_client.py

```diff
@@ -0,0 +1,480 @@
+# ------------------------------------
+# Copyright (c) Microsoft Corporation.
+# Licensed under the MIT License.
+# ------------------------------------
+
+import time
+
+from azure.core.tracing.decorator import distributed_trace
+
+from ._enums import LedgerUserRole, TransactionState
+from ._generated_ledger.models import ConfidentialLedgerQueryState
+from ._models import (
+    AppendResult,
+    Constitution,
+    Consortium,
+    ConsortiumMember,
+    EnclaveQuote,
+    LedgerEnclaves,
+    LedgerEntry,
+    LedgerUser,
+    TransactionReceipt,
+    TransactionStatus,
+)
+from ._shared import (
+    ConfidentialLedgerCertificateCredential,
+    ConfidentialLedgerClientBase,
+)
+
+try:
+    from typing import TYPE_CHECKING
+except ImportError:
+    TYPE_CHECKING = False
+
+if TYPE_CHECKING:
+    from azure.core.credentials import TokenCredential
+    from azure.core.paging import ItemPaged
+    from typing import Any, Union
+
+
+class ConfidentialLedgerClient(ConfidentialLedgerClientBase):
+    """A client for putting data into and querying data from the Confidential Ledger service.
+
+    The `transport` parameter is typically accepted by Azure SDK clients to provide a custom
+    transport stage in the pipeline. Since this client makes modifications to the default transport,
+    using a custom transport will override and remove the following functionality:
+        1) Authentication using a client certificate.
+        2) TLS verification using the Confidential Ledger TLS certificate.
+
+    :param str ledger_url: URL of the Confidential Ledger service.
+    :param credential: A credential object for authenticating with the Confidential Ledger.
+    :type credential: ~azure.confidentialledger.ConfidentialLedgerCertificateCredential
+    :param str ledger_certificate_path: The path to the ledger's TLS certificate.
+    :keyword api_version: Version of the Confidential Ledger API to use. Defaults to the most recent.
+    :type api_version: ~azure.confidentialledger.ApiVersion
+    """
+
+    def __init__(self, ledger_url, credential, ledger_certificate_path, **kwargs):
+        # type: (str, Union[ConfidentialLedgerCertificateCredential, TokenCredential], str, Any) -> None
+
+        super(ConfidentialLedgerClient, self).__init__(
+            ledger_url=ledger_url,
+            credential=credential,
+            ledger_certificate_path=ledger_certificate_path,
+            **kwargs,
+        )
+
+    @distributed_trace
+    def append_to_ledger(
+        self,
+        entry_contents,  # type: str
+        **kwargs,  # type: Any
+    ):
+        # type: (...) -> AppendResult
+        """Appends an entry to the Confidential Ledger.
+
+        :param entry_contents: Text to write to the ledger.
```

Comment:
tiny nit: It could be worthwhile to include an example of what `entry_contents` would look like

Reply:
Is the recommended approach a `samples` folder with ingestion into docstrings as suggested [here](https://review.docs.microsoft.com/en-us/help/contribute-ref/contribute-ref-how-to-document-sdk?branch=master#api-reference)?

Reply:
Yeah, that's def a good way to do it. [Here's](https://github.com/Azure/azure-sdk-for-python/blob/master/sdk/textanalytics/azure-ai-textanalytics/azure/ai/textanalytics/_text_analytics_client.py#L166) an example of it in code